post-types/templates/dataviz/dataviz-single-template.php

```diff
@@ -27,11 +27,14 @@
 <div class="chart_wrapper" style="width:<?php echo $viz_width; ?>;">
 	<h2><?php echo $post->post_title; ?></h2>
 	<?php
-		if (isset($atts["hide_description"]) && $atts["hide_description"] == true) {
+		if (isset($atts["hide_description"]) && $atts["hide_description"] == "true") {
```

Comment:
@DragonVirus this should not be a string, it should be a boolean . Suggestion:

```if (isset($atts["hide_description"]) && $atts["hide_description"] === true) {```

Also , see comment below